sips/minutes/_posts/2017-02-14-sip-minutes.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: sip-landing
+title: SIP Meeting Minutes - 14th February 2017
+---
+
+# Minutes
+
+The following agenda was distributed to attendees:
+
+| Topic | Reviewer |
+| --- | --- |
+| [SIP-XX: Improving binary compatibility with @stableABI](http://docs.scala-lang.org/sips/pending/binary-compatibility.html) | Dmitry Petrashko |
+| [SIP-NN - Allow referring to other arguments in default parameters](http://docs.scala-lang.org/sips/pending/refer-other-arguments-in-args.html) | Pathikrit Bhowmick |
+| [SIP 25 - @static fields and methods in Scala objects(SI-4581)](http://docs.scala-lang.org/sips/pending/static-members.html) | Dmitry Petrashko, Sébastien Doeraene and Martin Odersky |
+| [SIP-NN - Match infix & prefix types to meet expression rules](http://docs.scala-lang.org/sips/pending/make-types-behave-like-expressions.html) | Oron Port |
+
+Jorge Vicente Cantero was the Process Lead. Travis (@travissarles) was acting secretary of the meeting.
+
+## Date, Time and Location
+
+The meeting took place at 5:00pm Central European Time / 8:00am Pacific Daylight
+Time on Tuesday, 14th February 2017 via Google Hangouts.
+
+Minutes were taken by Travis Lee, acting secretary.
+
+## Attendees
+
+Attendees Present:
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Dmitry Petrashko (@DarkDimius)(https://github.com/DarkDimius), EPFL
+* Lukas Rytz ([@lrytz](https://github.com/lrytz)), Lightbend
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Independent
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), EPFL
+* Eugene Burmako ([@xeno-by](https://github.com/xeno-by)), EPFL
+* Jorge Vicente Cantero ([@jvican](https://github.com/jvican)), Process Lead
+
+Absent:
+* Heather Miller ([@heathermiller](https://github.com/heathermiller)), Scala Center
+* Josh Suereth ([jsuereth](https://github.com/jsuereth)),
+
+## Proceedings
+
+### Opening Remarks
+
+### SIP 25 - @static fields and methods in Scala objects(SI-4581)
+Jorge asks Dmitry to explain the biggest changes since the last proposal.
+The biggest changes were addressing the feedback of reader.
+**Dmitry** First, he covers whether the static annotation can behave like the tail recursive annotation, which doesn't actually impact compilation but only warns if it isn't possible to make something static. Dmitry doesn't think the static annotation should have the same semantics because it affects binary compatibility. Also, it depends on the superclass. If the superclass defined a field with the same name, the subclass can't have it. If we decide to make something static, we only be able to do it if the superclass doesn't have the fields with the same name. If static is done automatically and not triggered by the user we will enforce very strong requirements on superclass objects which is no-go.
```

Comment:
I think you should add a newline before this to break it off the previous paragraph.